ohmec.js

```diff
@@ -134,15 +134,17 @@ let geojson;
 function highlightFeature(e) {
   let layer = e.target;
 
-  layer.setStyle({
-    weight: 5,
-    color: '#666',
-    dashArray: '',
-    fillOpacity: 0.7
-  });
-
-  if (!L.Browser.ie && !L.Browser.opera && !L.Browser.edge) {
-    layer.bringToFront();
+  if (layer.feature.geometry.type !== "Point") {
+    layer.setStyle({
+      weight: 5,
+      color: '#666',
+      dashArray: '',
+      fillOpacity: 0.7
```

Comment:
this line is still tripping us up for Native entities

Reply:
Where are you seeing these lines tripping us up for Native entities? I'm not seeing this.

Reply:
> Where are you seeing these lines tripping us up for Native entities? I'm not seeing this.

Just that the fillOpacity takes the otherwise-transparent Native entities to opacity of 0.7 (on my side at least)

Reply:
Done